js/date-picker/js/lumx.date_picker_directive.js

```diff
@@ -111,6 +145,20 @@ angular.module('lumx.date-picker', [])
             }, 600);
         };
 
+        // month and year toggle activation based on flag
+        // if flag is true, year selection is displayed, else(default) monthly date selection
+        $scope.displayYear = function(flag) {
+            $scope.isYearSelected = flag;
+
+            var $innerListItem = $('.lx-date-picker__years--is-selected');
```

Comment:
We have to use `angular.element` instead of searching in `$element`. Reason being that, `$scope.openPicker()` attach the datepicker to `body` and hence `$element` now no more contain **lumx.date_picker.html** code.